src/runtime/import.cpp

```diff
@@ -453,7 +452,7 @@ Box* importModuleLevel(std::string* name, Box* globals, Box* from_imports, int l
     if (!parent)
         return NULL;
 
-    std::string _name = *name;
+    std::string _name = name;
```

Comment:
this one is necessary (we need the copy) because loadNext actually modifies _name.